subprojects/docs/docs.gradle

```diff
@@ -458,11 +458,7 @@ task editReleaseNotes() {
     }
 }
 
-task releaseNotesMarkdown(type: PegDown) {
-    group = "release notes"
-    markdownFile = file("src/docs/release/notes.md")
-    destination = new File(buildDir, "release-notes-raw/release-notes.html")
-}
+task releaseNotesMarkdown(type: PegDown, group: "release notes", constructorArgs: [ file("src/docs/release/notes.md"), new File(buildDir, "release-notes-raw/release-notes.html")])
```

Comment:
Nit: the space after `[` is unnecessary (or there needs to be a space before `]`, too).

Reply:
inconsistency is bad ☹️ .  fixed.